common/src/main/java/com/wynntils/features/user/DummyOverlayFeature.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.mojang.blaze3d.platform.Window;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.overlays.Overlay;
+import com.wynntils.core.features.overlays.annotations.OverlayHolder;
+import com.wynntils.core.features.overlays.annotations.RegisteredOverlay;
+import com.wynntils.mc.render.RenderUtils;
+import com.wynntils.utils.objects.CustomColor;
+
+@OverlayHolder
```

Comment:
Same as by Config (Old Configurable), this isn't needed and you can simply get all the declared classes and then get the superclass.

Reply:
This is used for explicit registration of overlays. Later, a feature will be able to hold Overlay, but not neccessary instantiate it.